README.md

```diff
@@ -21,16 +21,16 @@
 
 ## Features
 
-* Write slides in the **Markdown** with extended syntax.
+* Compose slides in **Markdown** with extended syntax.
```

Comment:
I'm not convinced by the 'compose'. By using the 'write', I wanted to emphasise that the slides are created in a text editor. It feels as though the 'compose' is taking the concept in another direction and further from the intent. However, I'm willing to be convinced that this is a good choice in this context. Though, in practice, I haven't heard anyone use the word 'compose' when working on slides.

Reply:
Different verbs go well with different objects.  "Write slides" may not be _syntactically_ wrong, but it definitely sounds awkward.  If you want to use the word "write", I would say something like "Create slides by simply writing in Markdown".

Reply:
I like your suggestion. Would you mind amending the PR please?

Reply:
@brandondrew Would you find a minute to update your PR?

Reply:
I'm not actually sure how to do that.  

GitHub makes it easy to submit PRs from forked repos directly within GitHub, and obviously I know how to submit PRs to repos I have access to from my laptop, but if I pull my fork down and make the changes locally, and force push them to my fork, will that force push the updates to the PR in your repo?

Reply:
Sorry if it should be obvious... it's almost time for bed in my time zone and I mainly need to be ready for a meeting first thing tomorrow morning, so I haven't got time to figure it out right now.